domain-entity/src/main/java/com/wednesday/template/domain/lastfm/Album.kt

```diff
@@ -0,0 +1,12 @@
+package com.wednesday.template.domain.lastfm
+
+data class Album(
+    val artist: String?,
+    val mbid: String?,
+    val name: String?,
+    val streamable: String?,
+    val smallImage: String?,
+    val mediumImage: String?,
+    val xlImage:String?,
+    val url: String?
```

Comment:
Why are all these nullable? We should be able to make sure that these are not null or else provide default values